controllers/passwords.js

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from '@prisma/client';
+import sendResponse from '../helpers/responseHelper.js';
+import createPasswordSchema from '../validationSchemas/passwordSchema.js';
+
+const prisma = new PrismaClient();
+
+export const handleGetPasswords = async (req, res) => {
```

Comment:
what does this method do?

Reply:
Fetch all the passwords from DB
Route: '/passwords'
Method: GET
We might alter it later. I made this to see if passwords are saving into DB or not.

Reply:
We have to remove this method later. Because we might not want to fetch passwords this way

Reply:
This can be a security flaw

Reply:
Yes, this is definitely gonna be removed. It is just used for testing purposes.